### PR TITLE
mgmt, enable live tests for ADF and DPS

### PR DESCRIPTION
**Pull request by weidongxu-microsoft**
DPS got genuine failure because they've got one API not properly deployed in ARM. Service fix rollout in 1/29.

# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [x] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).**

## [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#developer-guide)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#building-and-unit-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/datafactory/azure-resourcemanager-datafactory/pom.xml [hunk 1]
```diff
@@ -51,6 +51,12 @@
       <artifactId>azure-core-management</artifactId>
       <version>1.5.1</version> <!-- {x-version-update;com.azure:azure-core-management;dependency} -->
     </dependency>
+    <dependency>
```
Comment (haolingdong-msft) [inline sdk/datafactory/azure-resourcemanager-datafactory/pom.xml:54]: why we need junit-jupiter-engine?

**Review by haolingdong-msft — APPROVED** [review]

File: sdk/datafactory/azure-resourcemanager-datafactory/pom.xml (hunk 1 above)
Comment (weidongxu-microsoft) [reply sdk/datafactory/azure-resourcemanager-datafactory/pom.xml:54]: It is included in azure-core-test as test dependency, hence does not available to this lib via transient dependency.

**Pull request merged**